ProjFS.Mac/PrjFSKextLogDaemon/PrjFSKextLogDaemon.cpp

```diff
@@ -283,6 +287,18 @@ static bool TryFetchAndLogKextHealthData(io_connect_t connection)
     return true;
 }
 
+static void ReportDroppedKextMessages()
+{
+    uint32_t droppedMessageCount = atomic_exchange(&s_droppedMessageCount, 0U);
```

Comment:
This makes sense, we zero out s_droppedMessageCount and assign the current value to droppedMessageCount.  The convention may not be obvious to a kext newbe so consider a comment.